training/train_midi.py

```diff
@@ -39,10 +39,13 @@ def visualize_progress(generator: nn.Module, noise: torch.Tensor, epoch: int, ba
     # We can thus squeeze the first dimension to get a tensor of shape (channels, noise_size)
     fake_data = generator(noise).squeeze(0).detach().cpu().numpy()
 
-    dstart = fake_data[0, :]
-    duration = fake_data[1, :]
+    dstart = fake_data[0, :] * 48.916666666666515
+    duration = fake_data[1, :] * (99.45833333333331 - 0.0010416666666515084) + 0.0010416666666515084
```

Comment:
The values here are min/max values calculated during normalization, they are printed during preprocessing. You can check them when running `preprocess_maestro.py`